common/fixation_migrate.go

```diff
@@ -77,3 +113,26 @@ func fixationMigrate1to2(ctx sdk.Context, fs *FixationStore) error {
 	}
 	return nil
 }
+
+// fixationMigrate2to3: fix refcounts
+//   - instead of "EntryPrefix + fs.prefix" -> "fs.prefix + EntryPrefix"
+//   - instead of "EntryIndexPrefix + fs.prefix"(x2) -> "fs.prefix + EntryIndexPrefix"
+func fixationMigrate2to3(ctx sdk.Context, fs *FixationStore) error {
+	const (
+		// copy of old values
+		V2_EntryIndexPrefix string = "Entry_Index_"
```

Comment:
old values and new values are the same?

Reply:
in case the current value change in a newer version.